### DISCUSSION TITLE
AppArmor error message during installation

### DISCUSSION THREAD

**Discussion by kabelwalter** [nextcloud/all-in-one#2285] [Questions] [answered]
Hi, 
As a newcomer to Nextcloud, I would like to install AIO on my Proxmox. 
I get this error during the installation:
docker: Error response from daemon: AppArmor enabled on system but the docker-default profile could not be loaded: running `/usr/sbin/apparmor_parser apparmor_parser -Kr /var/lib/docker/tmp/docker-default581941599` failed with output: apparmor_parser: Unable to replace "docker-default".  Permission denied; attempted to load a profile while confined?

It is a fresh LXC container with Ubuntu 22.04.2 LTS and the packages updated/upgraded. 
The data directory is a ZFS mountpoint and Caddy is set up as reverse proxy.

What is my error?
Thank you for replies.

**Comment by szaimen** [comment, marked as answer]
Hi, I found this for you: https://www.google.com/search?q=+apparmor_parser%3A+Unable+to+replace+docker-default.+Permission+denied%3B+attempted+to+load+a+profile+while+confined%3F&biw=360&bih=617&ei=kJAuZOaZFeXlxc8P6fuJ6AU&oq=+apparmor_parser%3A+Unable+to+replace+docker-default.+Permission+denied%3B+attempted+to+load+a+profile+while+confined%3F&gs_lcp=ChNtb2JpbGUtZ3dzLXdpei1zZXJwEANKBAhBGABQhlhYjnFgp3doAHAAeACAAQCIAQCSAQCYAQCgAQHAAQE&sclient=mobile-gws-wiz-serp

**Reply by kabelwalter** [reply]
Ok - Thank you! If you know how, it's so easy:
sudo apt remove apparmor
And the NC AIO installation continues.
